apps/box/src/screens/Settings/ComponentGallery/UsageBarDemo.screen.tsx

```diff
@@ -0,0 +1,46 @@
+import {
+  FxBox,
+  FxReText,
+  FxSafeAreaBox,
+  FxSpacer,
+  FxText,
+} from '@functionland/component-library';
+import { UsageBar, UsageBarUsage } from '../../../components';
+import { HeaderText } from '../../../components/Text';
+import React from 'react';
+import { useDerivedValue, useSharedValue } from 'react-native-reanimated';
+
+const poolUsage: UsageBarUsage[] = [{ usage: 15000, color: 'pink' }];
+const personalUsage: UsageBarUsage[] = [
```

Comment:
possibly don't need to explicitly specify the `UsageBarUsage[]` type here as the `usages` prop would warn you if wrong

Reply:
It's kind of nice having it here. Without it, it allows you to add additional props to the object without complaining.